subjects/devops/count-files/README.md

```diff
@@ -0,0 +1,28 @@
+## Count files
+
+### Instructions
+
+Create a file `count-files.sh`, which will print the number **(and only the number)** of regular files and folders contained in the current directory and its sub-folders (the current directory must be included in the count).
+
+### Usage
+
+```console
+$ ./count-files.sh | cat -e
+12$
+$
+```
+
+### Hints
+
+Here are some Commands that can help you:
+
+- `find`. Find files or directories under the given directory tree, recursively.
+- `wc`. Count lines, words, and bytes.
```

Comment:
need to give an examples of wc command that can help in this case, lines, words, bytes

Reply:
done